### DISCUSSION TITLE
Cron Workflows with Hash for preventing to run at same time.

### DISCUSSION THREAD

**Discussion by gcezaralmeida** [argoproj/argo-workflows#6272] [Q&A]
Hi all, I would like to know if it is possible to avoid multiple cron workflows to run at the same time. For example: 
I have multiple workflows that run each 5 minutes and I would like than to start at different times. I know with jenkins there is a possibility to do that with H/5 * * * *. What I understood is that H is a hash that helps spread the tasks over time and don't let start at the same time avoiding resource utilization spikes.

**Comment by krrrr38** [comment]
ref https://github.com/argoproj/argo-workflows/pull/16604
